wikimedia/wikipedia-ios
# T374160: Profile view

**Phabricator:** 
https://phabricator.wikimedia.org/T374160

### Notes

### Test Steps
1. Open app, logged in or not, feel free to test both
2. Press little person icon in upper right hand corner
3. Slide down or press done to dismiss 
4. If logged in, ensure username is on top 
5. Check to ensure the proper layout, notifications number

### Screenshots/Videos

<img width="444" alt="Screenshot 2024-09-17 at 3 48 48 PM" src="https://github.com/user-attachments/assets/629341d5-f529-49be-84a3-b0e7c54d59f4">
